lesson-5/chess_board.js

```diff
@@ -0,0 +1,41 @@
+var $div = document.createElement('div');
+$div.id = 'chess-board';
+document.body.appendChild($div);
+
+var $chessBoard = document.getElementById('chess-board');
+
+letters = ['', 'A', 'B', 'C', 'D', 'E', 'F', 'G', 'H', '']
+numerals = ['8', '7', '6', '5', '4', '3', '2', '1']
+
+for (var i = 0; i < 10; i++) {
+    for (var j = 0; j < 10; j++) {
+        var $div = document.createElement('div');
+        if (i == 0 || i == 9) {
```

Comment:
Лучше использовать строгое сравнение
```suggestion
        if (i === 0 || i === 9) {
```